src/main/scala/io/smartdatalake/app/SmartDataLakeBuilder.scala

```diff
@@ -75,7 +75,7 @@ abstract class SmartDataLakeBuilder extends SmartDataLakeLogger {
 
   // read version from package manifest (not defined if project is executed in IntellJ)
   val appVersion: String = Option(getClass.getPackage.getImplementationVersion).getOrElse("develop")
-  val appName: String = getClass.getSimpleName.replaceAll("\\$$","") // remove $ from object name and use it as appName
+  val appType: String = getClass.getSimpleName.replaceAll("\\$$","") // remove $ from object name and use it as appName
```

Comment:
Can you give the reasoning behind the variable name change appName to appType ? I see sometimes it's still appName.
Didn't check everything, but I guess a few comments / references like here would also need to be changed.

Reply:
The applicationName is actionally given as parameter on the command line. It's confusing if we use it also here, so i was looking for a different name.
I'll check the other references.

Reply:
No additional wrong references found for appName with text search.